content/en/docs/instrumentation/python/automatic.md

```diff
@@ -28,8 +28,12 @@ with the agent. They should both produce the same results, demonstrating that
 the automatic instrumentation agent does exactly the same thing as manual
 instrumentation.
 
-To better understand auto-instrumentation, see the relevant part of both
-scripts:
+To better understand auto-instrumentation it's useful to know how OpenTelemetry achieves 
+the automatic instrumentation. It happens through monkey-patching which is done the
```

Comment:
Without defining what "monkey patching" is I think some folks will get confused.

Reply:
I believe most python readers would be familiar with the term.